### PR TITLE
Fran's README.md file w/ brief bio in markup

### PR DESCRIPTION
**Pull request by FrannyMendesLevitin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,28 @@
...
+* Painting
+
+
+**ANALOG VIDEO MIXER I hope to build myself**
+![](visualCortex.png)
+
+#### Reflection on workflow  
+I've had some experience using git and github, so I used the terminal for parts of the process.  I first forked the repository from the class's github page.  Next, I opened the terminal on my local machine and cloned the form by executing the command "git clone <HTTP repo on github>".  Next I opened up the file README.md in RStudio and creted a new project from my existing directory.  I made changes localy to the file in RStudio.  Finally, when I finished the assignment I commited my changes and pushed to my forked repository.  
+
+
```
Comment (ifarah) [inline README.md:26]: **Coding style**  +
 **Coding strategy**   +
 **Presentation**  +
 **Achievement, creativity** +
 **Ease of access** +

It was great how you you effectively used five of the five style elements, photo, italics, headers, list, and link. It was nice getting to read your biography, and knowing a bit about you and what you do.

**Review by ifarah — COMMENTED** [review]
Peer review

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.Rproj.user
```
Comment (limchengyee) [inline .gitignore:1]: Good use of .gitignore

File: README.md [hunk 3]
```diff
@@ -1,2 +1,28 @@
...
+### Art Practice Activities
+* Scavenging   
+* Programing  
+* Hacking  
+* Live Audio/Visual performance
+* Painting
+
+
+**ANALOG VIDEO MIXER I hope to build myself**
+![](visualCortex.png)
```
Comment (limchengyee) [inline README.md:21]: Will be a good idea to create a sub folder for images in the future for repository organization, when we submit include more images in our submission

**Review by limchengyee — COMMENTED** [review]
Evaluation of HW01:

Coding Style: +
Coding Strategy: +
Presentation: +
Achievement, Creativity: +
Ease of Access: +

Reviews:
Good job! This assignment is aesthetically pleasing, clear and concise. It also demonstrated his competence in Rmarkdown (i.e. bold, italics, hyperlinks, pictures, lists and headers) and his experience with github.

File: README.md (hunk 3 above)
Comment (bensoltoff) [reply README.md:21]: Agreed, though for simple repos like this one it works fine

File: README.md [hunk 4]
```diff
@@ -1,2 +1,28 @@
...
+---
+title: "Fran's first markdown" 
+author: "Frances Mendes Levitin"
+---
+
+
+# hw01: Repository for first CFSS homework assignment
+
+## Who
+My name is Frances Mendes Levitin.  I am a first year student in the *Department of Visual Arts* (DoVA) at University of Chicago. To see more art go to by website by clicking this [link](http://45.55.61.6:3000/).
```
Comment (bensoltoff) [inline README.md:10]: Get yourself a [domain](https://domains.google/) for your website! Or alternatively, is it something that can be hosted on [GitHub Pages](https://pages.github.com/)?

**Review by bensoltoff — COMMENTED** [review]
Evaluation HW-01 for Frances Levitin
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |         x     |            |
| **Coding strategy**         |           |        x      |            |
| **Presentation**            |    x       |              |            |
| **Achievement, creativity** |       x    |              |            |
| **Ease of access**          |     x      |              |            |

Remarks:

* Nice submission for the first assignments. Includes all the major features I was looking for
* In the future, include your reflections as a comment on the pull request, rather than in the readme for the repo

**Pull request closed without merging**